### PR TITLE
Issue #6586: aligned javadoc/xdoc for FallThrough

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6586 (part of #5750)

Second commit to force a whitespace after period/comma/etc.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java [hunk 1]
```diff
@@ -29,14 +29,42 @@
 import com.puppycrawl.tools.checkstyle.utils.CommonUtil;
 
 /**
- * Checks for fall through in switch statements
- * Finds locations where a case <b>contains</b> Java code -
- * but lacks a break, return, throw or continue statement.
+ * <p>
+ * Checks for fall-through in {@code switch} statements.
+ * Finds locations where a {@code case} <b>contains</b> Java code but lacks a
+ * {@code break},{@code return},{@code throw} or {@code continue} statement.
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java]: The system is removing the space between `,` and `{`.
Can this be fixed?

Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java]: Done. Moved to separate commit due to significant amount of changes
b833ad1fe78985d703b8db48cbcb73351695cc75

Comment (rnveach) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java]: Thanks.

**Comment by rnveach** [comment]
@pbludov Somehow you changed the results of pitest?
I am not seeing any failures in master. https://circleci.com/gh/checkstyle/checkstyle/tree/master

**Comment by pbludov** [comment]
> @pbludov Somehow you changed the results of pitest?

This is surprising failure for me.

**Comment by rnveach** [comment]
I will try restarting it.

**Comment by rnveach** [comment]
It passed now. Weird.

**Pull request merged**